metricbeat/module/http/_meta/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# Tomcat is started to fetch Jolokia metrics from it
```

Comment:
Any suggestions? I reused the container Jolokia container as it was already there, so that no additional image needs to be build and an additional to be started.